app/src/main/res/values/strings.xml

```diff
@@ -29,7 +29,8 @@
         Paulo Schopf\n
         Joaquim Pedroto Costa\n
         @madcrow\n
-        @Tubix
+        @Tubix\n
+        ceskyDJ
```

Comment:
Here if you add your GitHub name, please add at sign. You can put _@ceskyDJ_ or, if prefer without at, your name _Michal Šmahel_.

Reply:
Ok, I changed it in commit ae3cd7bc.